FakebookNotifications/FakebookNotifications.DataAccess/NotificationsContext.cs

```diff
@@ -0,0 +1,61 @@
+﻿using FakebookNotifications.DataAccess.Models;
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FakebookNotifications.DataAccess
+{
+    public class NotificationsContext
```

Comment:
the class should go through TDD, maybe not every part of it can be unit tested but some parts can, so this is the wrong kind of PR to add it in.